layers/descriptor_sets.cpp

```diff
@@ -686,6 +710,8 @@ bool cvdescriptorset::ValidateDescriptorSetLayoutCreateInfo(
                             "for %s since descriptorBindingInlineUniformBlockUpdateAfterBind is not enabled.",
                             i, string_VkDescriptorType(binding_info.descriptorType));
                     }
+#endif
+#if !defined(VULKANSC)
```

Comment:
This should probably be based on the extensions? (NV_ray_tracing, KHR_acceleration_structure)

Reply:
ack